lib/embedded_ansible.rb

```diff
@@ -8,6 +8,7 @@ class EmbeddedAnsible
   ANSIBLE_ROLE           = "embedded_ansible".freeze
   SETUP_SCRIPT           = "ansible-tower-setup".freeze
   SECRET_KEY_FILE        = "/etc/tower/SECRET_KEY".freeze
+  SETTING_FILE           = "/etc/tower/settings.py".freeze
```

Comment:
I think `SETTINGS_FILE` would be a better constant name.

Reply:
This is for socks5 proxy. If leave empty or "http" a http proxy is used. If "socks5", in lib/vmdb/utils.rb: https://github.com/ManageIQ/manageiq/blob/master/lib/vmdb/util.rb#L3
VMDB::Util.http_proxy_uri will return a socks5 proxy. Both http proxy and socks5 proxy works as value of `http_proxy` environment variable

Reply:
OK!